### ISSUE TITLE
Feature Request: list_slice as alias of array_slice

### ISSUE THREAD

**Issue by Alex-Monahan** [duckdb/duckdb#2795]
Hi Folks!

As I was working on the nested types documentation, array_slice was the only gap in our list naming scheme that forced us to use a function starting with "array_" and not "list_". (all others had a "list_" alias or alternative like len()). 

Would you be open to adding a list_slice function that is an alias of array_slice? I don't think other systems implement such a function, but I think that's because other systems tend to use array instead of list.

Thanks!
-Alex

**Comment by makalaaneesh** [comment]
Hi, I'd like to pick this up!

I'm assuming what is requested is just to have `list_slice` be an alias to `array_slice`?

**Comment by lnkuiper** [comment]
Correct! Should be an easy fix. Could you add a test case to the array slice tests that tests the new alias as well?
